src/odemis/acq/stream/_helper.py

```diff
@@ -669,6 +669,41 @@ def __init__(self, name, detector, dataflow, emitter, analyzer=None, **kwargs):
 
     # onActive & projection: same as the standard LiveStream
 
+    # overwrite fct _is_active_setter in _base.py
+    def _is_active_setter(self, active):
+        # call fct _is_active_setter in _base.py
+        active = super(ARSettingsStream, self)._is_active_setter(active)
+        if active:
+            self._linkHwAxes()
+        else:
+            self._unlinkHwAxes()
+        return active
+
+    def _linkHwAxes(self):
+        """"subscribe polarization VA: link VA to hardware axis"""
+        try:
+            logging.debug("Moving polarization analyzer to position %s.", self.polarization.value)
+            f = self.analyzer.moveAbs({"pol": self.polarization.value})
+            f.result()
+        except Exception:
+            logging.exception("Failed to move polarization analyzer.")
+        self.polarization.subscribe(self._onPolarization)
```

Comment:
You could add a note/TODO that ideally, it would also listen to the `analyzer.position`, and update the `polarization` VA whenever the axis has moved.